setup.cfg

```diff
@@ -1,3 +1,60 @@
+[metadata]
+name = ewah_bool_utils
+version = 0.1.0
+description = EWAH Bool Array utils for yt
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+url = https://github.com/themousepotato/ewah_bool_utils
+author = Navaneeth Suresh
+author_email = navaneeths1998@gmail.com
+license = BSD-3-Clause
+license_file = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: BSD License
+    Natural Language :: English
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+keywords = ewah_bool_utils
+
+[options]
+packages = find:
+install_requires =
+    numpy>=1.10.4
+python_requires = >=3.5
+include_package_data = True
+zip_safe = False
+
+[options.packages.find]
+include =
+    ewah_bool_utils
+    ewah_bool_utils.*
+
+[options.extras_require]
+dev =
+    Cython>=0.29.21,<3.0
+    Sphinx==1.8.5
+    bump2version==0.5.11
+    coverage==4.5.4
+    flake8==3.7.8
+    pytest==4.6.5
+    pytest-runner==5.1
+    tox==3.14.0
+    twine==1.14.0
+    watchdog==0.9.0
+    wheel==0.33.6
```

Comment:
I think Sphinx, coverage, tox, watchdog are unused.
twine and wheel could probably be left out too because I suspect they are only supposed to be used for publication, which doesn't seem to be setup already. Another time probably !